### PR TITLE
Add example that shows how to use managed code from a native app with Xaml Islands

### PR DESCRIPTION
**Pull request by ocalvo**
Add example that shows how to use managed code from a native app with Xaml Islands.

### CODE REVIEW

File: 1903_Samples/CppWinRT_Win32_App/SampleCppApp.Package/Images/dummy.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+dummy
```
Comment (marb2000) [inline 1903_Samples/CppWinRT_Win32_App/SampleCppApp.Package/Images/dummy.txt]: Why do you need a text file into the image folder?

File: 1903_Samples/CppWinRT_Win32_App/SampleCppApp/SampleApp.cpp [hunk 2]
```diff
@@ -104,11 +106,17 @@ class MyWindow : public DesktopWindowT<MyWindow>
...
-        m_xamlBt1.Height(ButtonHeight / dpi);
-        m_xamlBt1.Width(ButtonWidth / dpi);
-        m_xamlBt1ClickEventRevoker = m_xamlBt1.Click(winrt::auto_revoke, { this, &MyWindow::OnXamlButtonClick });
-        m_hWndXamlButton1 = wil::unique_hwnd(CreateDesktopWindowsXamlSource(WS_TABSTOP, m_xamlBt1));
+
+        m_managedControl = winrt::SampleLibraryCS::CustomUserControl();
+        m_managedControl.Height(ButtonHeight / dpi);
+        m_managedControl.Width(ButtonWidth / dpi);
+        m_hWndXamlButton1 = wil::unique_hwnd(CreateDesktopWindowsXamlSource(WS_TABSTOP, m_managedControl));
+
```
Comment (marb2000) [inline 1903_Samples/CppWinRT_Win32_App/SampleCppApp/SampleApp.cpp]: OK. This is very straightforward. I mean, it's the CLR load automatically?

File: 1903_Samples/CppWinRT_Win32_App/SampleLibraryCS/Package.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+using Microsoft.ReactNative;
+using Microsoft.ReactNative.CSharp;
+
```
Comment (marb2000) [inline 1903_Samples/CppWinRT_Win32_App/SampleLibraryCS/Package.cs]: Why do you need this Package.cs? Why do we need ReactNative?

**Review by marb2000 — COMMENTED** [review]
Some questions about this magical example that requires basically no-work.

File: 1903_Samples/CppWinRT_Win32_App/SampleCppApp.Package/Images/dummy.txt (hunk 1 above)
Comment (ocalvo) [reply 1903_Samples/CppWinRT_Win32_App/SampleCppApp.Package/Images/dummy.txt]: There will be a build break in the WAP project if this is not pressent

File: 1903_Samples/CppWinRT_Win32_App/SampleLibraryCS/Package.cs (hunk 3 above)
Comment (ocalvo) [reply 1903_Samples/CppWinRT_Win32_App/SampleLibraryCS/Package.cs]: Upps,

**Pull request merged**